src/SDDPoptimize.jl

```diff
@@ -128,23 +135,18 @@ function run_SDDP!(model::SPModel,
         push!(stats.exectime, toq())
         push!(stats.upper_bounds, upb)
 
-        if (display > 0) && (iteration_count%display==0)
-            if upb < Inf
-                println("Pass number ", iteration_count,
-                    "\tUpper-bound: ", upb,
-                    "\tLower-bound: ", round(stats.lower_bounds[end], 4),
-                    "\tTime: ", round(stats.exectime[end], 2),"s")
-             else
-                println("Pass number ", iteration_count,
-                    "\tLower-bound: ", round(stats.lower_bounds[end], 4),
-                    "\tTime: ", round(stats.exectime[end], 2),"s")
-             end
+        if (verbose > 0) && (iteration_count%verbose==0)
+            print("Pass number ", iteration_count)
+            (upb < Inf) && print("\tUpper-bound: ", upb)
+            println("\tLower-bound: ", round(stats.lower_bounds[end], 4),
+                "\tTime: ", round(stats.exectime[end], 2),"s")
         end
 
     end
-
-    # Estimate upper bound with a great number of simulations:
-    if (display>0) && (param.compute_upper_bound != 0)
+    
+    ##########
+    # Estimate final upper bound with a great number of simulations:
+    if (verbose>0) && (param.compute_upper_bound != 0)
```

Comment:
I am lost about what param.compute_upper_bound is exactly supposed to be : 
+ 0 -> never computed
+ -1 -> computed at the end with param.monteCarloSize scenarios ? more ?
+ 10 -> computed every 10 iterations with a fixed set of param.monteCarloSize scenarios, and at the end with xxx scenarios ?

I would prefer to default to (i.e. compute_upper_bound = 0) the case where we compute the upb only at the end.